### DISCUSSION TITLE
3rd party devices

### DISCUSSION THREAD

**Discussion by ntbritton** [leikoilja/ha-google-home#687] [Q&A]
I was under the impression that this would collect devices added to google home, including non-google devices, is that a correct assumption?

When i look at the entities, i dont see my roku's or wyze sensors, that i was expecting to gather.

**Comment by leikoilja** [comment]
No, this integration takes care of communicating with Google Home (voice assistant) enabled devises. You are probably looking to connect google assistant enabled devices like your Roku/Wyze etc to HA? If so, this is probably what you are looking for https://www.home-assistant.io/integrations/google_assistant/
